Removing SfB as unsupported

CHDAFNI-MSFT opened the pull request:
SfB now supports Modern Authentication and is the default
https://technet.microsoft.com/en-us/library/mt803262.aspx

azurecla commented:
Hi __@diegoviso__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

CHDAFNI-MSFT committed:
Removing SfB as unsupported

SfB now supports Modern Authentication and is the default
https://technet.microsoft.com/en-us/library/mt803262.aspx
